index.js

```diff
@@ -354,20 +354,21 @@ program
   });
 
 /**
- * Creates a new script project. The project title is optional.
+ * Creates a new script project. The project title is optional. The project parent Id is optional.
+ * ParentId: The Drive ID of a parent file that the created script project is bound to. This is usually the ID of a Google Doc, Google Sheet, Google Form, or Google Slides file. If not set, a standalone script project is created.
  * @example `create "My Script"`
```

Comment:
Can you please add another line to include an example with a scriptParentId?

Reply:
Yes, I will add an example soon